backend/src/test/java/com/ody/route/controller/ApiCallControllerTest.java

```diff
@@ -0,0 +1,33 @@
+package com.ody.route.controller;
+
+import com.ody.common.BaseControllerTest;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ApiCallControllerTest extends BaseControllerTest {
+
+    @DisplayName("ODsay API 호출 횟수 조회에 성공하면 200 응답을 반환한다")
+    @Test
+    void countOdsayApiCall() {
+        RestAssured.given().log().all()
+                .when()
+                .get("/admin/api-call/count/odsay")
+                .then()
+                .log()
```

Comment:
합의 했던 컨벤션에 따라 한줄에 써도 좋을 것 같습니다!
```suggestion
                .then().log().all()
```